plugins/node/opentelemetry-instrumentation-express/test/hooks.test.ts

```diff
@@ -169,4 +171,97 @@ describe('ExpressInstrumentation hooks', () => {
       );
     });
   });
+
+  describe('request hooks', () => {
+    let server: http.Server;
+    let port: number;
+    let rootSpan: Span;
+
+    beforeEach(async () => {
+      rootSpan = tracer.startSpan('rootSpan');
+
+      const httpServer = await serverWithMiddleware(tracer, rootSpan, app => {
+        app.get('*', (req, res) => {
+          res.send('ok');
+        });
+      });
+      server = httpServer.server;
+      port = httpServer.port;
+    });
+
+    afterEach(() => {
+      server.close();
+    });
+
+    it('should call requestHook when set in config', async () => {
+      const requestHook = sinon.spy((span: Span, info: ExpressRequestInfo) => {
+        span.setAttribute('http.method', info.request.method);
```

Comment:
nit: prefer to use the enum: `SemanticAttributes.HTTP_METHOD` (here and in other places)